src/Organisms/Modal/ModalBackground.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+import PropTypes from 'prop-types'
+import * as React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: ${(props) => props.theme['modalBackgroundColor']};
+  z-index: 2000;
```

Comment:
can we remove that and let the consumers to set their own z-index?

Reply:
well, I anyway need a z-index for the background. I can add that as a prop with a default value instead

Reply:
removed all the z-indexes at the end